pkg/cmd/kafka/update/update.go

```diff
@@ -308,3 +314,13 @@ func getElementValue(v reflect.Value) reflect.Value {
 	}
 	return v
 }
+
+// get the true value from a reflect.Value for KafkaUpdateRequest struct
+// to do: Needs update once reauthentication_enabled is incorporated
+func getUpdateObjValue(v reflect.Value) reflect.Value {
+	if v.Kind() == reflect.Struct {
+		vstruct := v.Field(0)
```

Comment:
A better waywould be to get the `value` field instead of index. Do we need to iterate through this struct as well?

Reply:
This should work:

```suggestion
		vstruct := v.FieldByName("value")
```

This should suffice, no need to iterate.

Though I'm starting to think going forward with reflection on this is a bad idea, as we are making assumptions that are tightly coupled with the SDK, which could introduce breaking changes in the future by changing the name from "value" and since this is not strongly typed, we would not catch it..

For now, once you update with the suggestion above, we can merge as it is blocking any releases. Could you create a follow up issue to address this? We can simplify it by operating directly on the SDK model properties instead.

Reply:
Done!
Ready for review.